Doc/cmus.txt

```diff
@@ -1101,7 +1101,7 @@ repeat (false)
 repeat_current (false)
 	Repeat current track forever.
 
-replaygain (disabled) [track, album, track-preferred, album-preferred]
+replaygain (disabled) [track, album, track-preferred, album-preferred, smart]
 	Enable Replay Gain.
```

Comment:
It'd probably be a good idea to describe what `smart` does here since it's not as obvious as the others.

Reply:
@Allavaz can you please add the documentation for the new option?

Reply:
Just pushed a commit trying to explain what it does but I'm not sure if it's clear enough. Let me know if it's confusing or anything.